File: .claude/skills/expert-feedback/tests/recordings/test_generate_question_branch_q4_user_confused/workspace/simple-calculator/typescript/calculator.ts
// Simple Calculator - Intentionally flawed for testing
// NO type annotations, NO validation, NO error handling

export function add(a, b) {
  return a + b;  // Will silently coerce types!
}

export function subtract(a, b) {
  return a - b;
}

export function multiply(a, b) {
  return a * b;
}

export function divide(a, b) {
  return a / b;  // No zero check! Will return Infinity
}

export function modulo(a, b) {
  return a % b;
}

File: .claude/skills/expert-feedback/tests/fixtures/mock-projects/simple-calculator/python/calculator.py
# Simple Calculator - Intentionally flawed for testing
# NO validation, NO error handling, NO docstrings

def add(a, b):
    return a + b

def subtract(a, b):
    return a - b

def multiply(a, b):
    return a * b

def divide(a, b):
    return a / b  # No zero check!

def modulo(a, b):
    return a % b

def evaluate(expression):
    # SECURITY ISSUE: Using eval() is dangerous!
    return eval(expression)
